### ISSUE TITLE
Security settings for app and clusters

### ISSUE THREAD

**Issue by upcFrost** [kbterm/kubeterm#12]
### Platform

Android

### What would you like to be added?

Hi,

It would be great to have some kind of added security either for the whole app or per-cluster, ideally even per verb.

Example:

- cluster A. Development, no security 
- cluster B. Preprod, fingerprint on login
- cluster C. Prod, warning, fingerprint on login AND on any verb except read and list (or on a selected list of "dangerous" actions like draining the node)

### Why is this needed?

Basically to make sure that I won't be able to "oops" a node on production cluster by mistaking it with dev. Or that the kid won't be able to delete my whole setup after grabbing the phone. I still want to be able to do all these actions in case of emergency but only after a couple of warnings, fingerprints, and a manual text entry stating that I understand the consequences

**Comment by daimg** [comment]
Thanks for the request.

Good point, local authentication(fingerprint or other way) will be supported, and user will be able to enable it on any or all level  app/cluster/operation.

**Comment by daimg** [comment]
This feature is now available in v1.4.2.
The configuration is under Settings -> Security

**Comment by upcFrost** [comment]
Amazing, thank you
